modules/gitops-configuring-argo-cd-oidc.adoc

```diff
@@ -34,7 +34,8 @@ $ oc edit secret openshift-gitops-secret -n <namespace>
 ----
 yaml apiVersion: v1 
 kind: Secret 
-metadata: name: argocd-secret 
+metadata:
```

Comment:
@boczkowska, do you know who from QE can confirm this change? Thanks!

Reply:
@wtam2018 @iam-veeramalla @amitkrout can one of you take a look and verify this, please?

Reply:
Looks good to me

Reply:
Thank you, Abhishek!

Reply:
/lgtm

Reply:
Looks good to me. Thanks

Reply:
@jboxman, can you also fix L35. Fix `yaml apiVersion: v1` -> `apiVersion: v1`

Reply:
@shubhamagarwal19 done thanks!